### DISCUSSION TITLE
Button inside Custom Menu not focusable

### DISCUSSION THREAD

**Discussion by imthiyasm** [JedWatson/react-select#5258] [Q&A] [answered]
Hi,

I have created a button(Change data) inside the customMenu component. But, the focus/interactable is not happening for the button when pressing the tab key from the SelectMenu. Anyone can help to resolve this issue?

For example: https://codesandbox.io/s/custom-react-select-menu-component-git-forked-8jd0tz

**Comment by manjushsh** [comment, marked as answer]
@imthiyasm  [Code Sandbox](https://codesandbox.io/s/custom-react-select-menu-component-git-forked-uxiu9q?file=/src/custom-select-menu/custom-select.jsx:1390-1396)

**Reply by imthiyasm** [reply]
@manjushsh, Thank you for your great suggestion! And need one more help from the A11y requirements it should be focused when pressing the tab key from the SelectMenu to the first option and it should move to the button(Change data)?

Appreciate your help in advance!

**Reply by manjushsh** [reply]
Isn't first option always on focus?
lemme check..

**Reply by imthiyasm** [reply]
@manjushsh, No, it's not moving focus to the first option. Thank you!

**Reply by imthiyasm** [reply]
@manjushsh, Did you get a chance to look into this issue?

**Reply by manjushsh** [reply]
@imthiyasm ya. but focus isn't working properly..

**Reply by saipraveen-ch** [reply]
Hi, I'm also having a similar requirement when pressing the tab key, it should go to the focus from selectMenu to the first option and the CTA button. Can anyone please suggest to me how to implement this?

CC: @JedWatson @manjushsh
